checkout-app/src/app/home/home.component.scss

```diff
@@ -0,0 +1,38 @@
+h2 {
+  width: 100%;
+  padding-bottom: 30px;
+}
+
+#basket-of-groceries {
+  width: 80%;
+  // max-width: 200px;
+}
+
+.fad-btn {
+  margin-right: 20px;
+}
```

Comment:
ideally this should be bootstrap instead of being custom defined, so we can just override with our preferences. OR, all other buttons across the dashboard and other elements need to be reconfigured to use this